skipper.go

```diff
@@ -352,10 +352,13 @@ type Options struct {
 	// PluginDir defines the directories to load plugins from
 	PluginDirs []string
 
-	// FilterPlugins loads additional filters from modules
+	// FilterPlugins loads additional filters from modules. The first value in each []string
+	// needs to be the plugin name (as on disk, without path, without ".so" suffix). The
+	// following values are passed as arguments to the plugin while loading, see docs/plugins.md
```

Comment:
docs/plugins.md ?

Please create dev docs here in godoc and if you like to link the new docs: 

1. mkdocs.yml to create a new page
2. create the new page in docs/plugins.md
3. link base: https://zalando.github.io/skipper/ path as defined in mkdocs.yaml

Reply:
fixed